src/main/java/uk/gov/justice/digital/delius/data/api/Attendance.java

```diff
@@ -0,0 +1,42 @@
+package uk.gov.justice.digital.delius.data.api;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import io.swagger.annotations.ApiModelProperty;
+import java.time.LocalDate;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+
+@Data
+@Builder
+@AllArgsConstructor
+public class Attendance {
+
+    @ApiModelProperty(required = true)
+    private final Long contactId;
+
+    @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "yyyy-MM-dd")
+    @ApiModelProperty(required = true)
+    private final LocalDate attendanceDate;
+
+    private final boolean attended;
```

Comment:
Maybe add a few APIModelProperty with a description of what the values means and ideally any examples

Reply:
I agree but looked around other objects and on some, such as Conviction, see absolutely no annotations. Will do though !

Reply:
sure, no problem